patches/node/remove_makeexternal_case_for_uncached_internal_strings.patch

```diff
@@ -0,0 +1,51 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Tue, 8 Dec 2020 19:55:41 -0800
+Subject: Remove MakeExternal case for uncached internal strings
+
+https://chromium-review.googlesource.com/c/v8/v8/+/2565511
+
+V8 removed a case where we can make an uncached internal string
+through MakeExternal - to fix this we make the strings bigger so as to
+make then internal & external & not uncached (i.e. cached).
```

Comment:
Is this upstreamed?

Reply:
Yes: https://github.com/nodejs/node/pull/36451